CHANGELOG.md

```diff
@@ -4,8 +4,10 @@ This document describes changes between each past release.
 
 ## 6.1.2 (unreleased)
```

Comment:
Moving from `setup.cfg` to `pyproject.toml` should probably be considered a breaking change. I agree with a previous comment of yours (where you told it would be better to not do things in this order and issue the quick fix first), but it's also probably too late now.

To me, it's at least a new feature (so updating the minor, to 6.2.0), but probably a breaking change, so we should update the major (7.0.0)